incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,40 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates all objects of a [specified type](how-to.md/#admin-resources) 
```

Comment:
No slash between the filename and the anchor: `how-to.md#admin-resources`. Although it seems to work anyway on Chrome.

Also the formatting still doesn't match what's in the doc - e.g. the word "all" should be emphasized as `_all_` (looks like: _all_).